BreadPlayer.Views.UWP/Core/SharedLogic.cs

```diff
@@ -47,7 +51,26 @@ public SharedLogic()
             InitializeCore.IsMobile = ApiInformation.IsApiContractPresent("Windows.Phone.PhoneContract", 1);
 
             InitializeCore.IsMobile = Window.Current?.Bounds.Width <= 600;
+
+            MediaDevice.DefaultAudioRenderDeviceChanged += OnDefaultAudioRenderDeviceChanged;
```

Comment:
Oh, I see, that's why *MediaDevice.DefaultAudioRenderDeviceChanged* fired multiple times... 😅